frontend/src/pages/uid.tsx

```diff
@@ -0,0 +1,99 @@
+//Chakra Imports
+import {
+	Box,
+	Flex,
+	Text,
+} from "@chakra-ui/react";
+import { GetStaticPropsContext } from "next";
+import { NextSeo } from "next-seo";
+import Announcement from "src/components/site/Announcement";
+// Server side
+import { markdownSSR } from "src/mdx-helpers/ssr";
+import { MarkdownContent } from "src/mdx-helpers/types";
+//Styles
+import heroStyles from "../styles/Hero.module.css";
+
+const crypto: Crypto | undefined = self.crypto ?? (self as { msCrypto?: Crypto }).msCrypto; // eslint-disable-line no-restricted-globals
+
+const getRandomBytes: (n: number) => Uint8Array = (crypto ?
+  function () { // Browsers
+    return function (n: number): Uint8Array {
+      var a = new Uint8Array(n);
```

Comment:
Use `const` instead of `var`.
```suggestion
      const a = new Uint8Array(n);
```